### PR TITLE
Update Readme.md with more info

### PR DESCRIPTION
**Pull request by jarrettv**
Improve Readme with more information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+* No separate Z-endstop required
+* Crash protection
+
+## Requirements
+
+* Bed must be stable for probing force of 500-800 grams
+  * Switchwire, V1.8, Legacy, V0 beds are NOT rigid enough
+* Mounts to MGN-12H X-axis
+* Front mounted extruder (Clockwork2, LGX, Galileo)
+* Accuracy depends on good mechicanical condition
```
Comment (ScottSmall) [inline README.md:21]: s/mechicanical/mechanical/

File: README.md [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+* Will the hartk 2 piece board work with this? Yes, but it requires a slight modification. See discord pin.
+* Can I use an optical sensor not specified in the BOM? Not recommended, the team spent considerable time validating sensors in BOM.
+* Can I use micro-switch instead? Not recommended, but check here for more details: [Unklicky Tap](https://github.com/majarspeed/Unklicky/tree/main/Unklicky_TAP)
+* Can I use Tap with kinematic bed mounts? Most likely yes if your bed is rigid up to 800 grams of force.
+
+## (Bonus Points) Data Science
+
+Tap engineering is backed by data science. If you want to dive deeper check out the following resources:
+
+* [Probe accuracy across thermal envelop](https://github.com/KiloQubit/probe_accuracy)
```
Comment (ScottSmall) [inline README.md:48]: s/envelop/envelope/

File: README.md [hunk 3]
```diff
@@ -1,3 +1,58 @@
-Voron Tap
+# Voron Tap
+
+![Voron-Tap3](https://user-images.githubusercontent.com/219295/206089801-1c3b0685-ac0b-460a-9027-2fd87e1cb79e.gif)
```
Comment (KiloQubit) [inline README.md:3]: Can you add this GIF to the repository and reference it here, instead of using githubusercontent?

**Review by KiloQubit — CHANGES_REQUESTED** [review]

**Comment by KiloQubit** [comment]
@Badnoob-327 you might want to look over this as well.

File: README.md (hunk 3 above)
Comment (jarrettv) [reply README.md:3]: Yes, if you prefer. However, I followed github's recommended approach here: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files

Comment (KiloQubit) [reply README.md:3]: That is talking about images in issues and pull requests, not images in the repository readme.  Images in the repository readme should ideally be relative links to images in the same repository, as documented here

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files

This makes it easier for clones or branches to modify the image for example, or to get a local copy of the image for viewing offline.

Comment (jarrettv) [reply README.md]: done

**Review by KiloQubit — APPROVED** [review]
Looks good from the repository perspective, needs @Badnoob-327 approval for contents.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,58 @@
...
+* Crash protection
+
+## Requirements
+
+* Bed must be stable for probing force of 500-800 grams
+  * Switchwire, V1.8, Legacy, V0 beds are NOT rigid enough
+* Mounts to MGN-12H X-axis
+* Front mounted extruder (Clockwork2, LGX, Galileo)
+* Accuracy depends on good mechanical condition
+* Must have 5V available or special 24V PCB
```
Comment (tanaes) [inline README.md:22]: Perhaps change to 'available at toolhead or use a special 24V sensor PCB' to reduce potential ambiguity?

File: README.md [hunk 5]
```diff
@@ -1,3 +1,58 @@
...
+1. Update your `printer.cfg` as recommended in [Tap Klipper Instructions](config/tap_klipper_instructions.md)
+2. Home Z and test virtual Z endstop by lifting tool-head
+3. Heat soak your machine and run a couple `probe_accuracy samples=100` to "break-in" your probe
+4. Run a few more `probe_accuracy` checks (default of 10 probes)
+
+For well-built machines you can expect to see between 0.0000 and 0.0008 standard deviation.
+
+## FAQs
+
+* Will it hurt my print surface? No, as probing temp is restricted to 150C. The team recommends spray coated spring steel. Users of PEI applied surfaces may find minor indent if continously probing the same spot.
```
Comment (tanaes) [inline README.md:39]: Suggest:
```* Will it hurt my print surface? No, as probing temp is restricted to 150C. The team recommends powder coated PEI spring steel build plates. Users of smooth PEI surfaces may find minor burnish marks if continuously probing the same spot.```

**Review by tanaes — COMMENTED** [review]
Just a couple places i thought could be clarified

**Pull request merged**